mitxgraders/formulagrader/integralgrader.py

```diff
@@ -537,3 +551,326 @@ def raw_integrand(x):
+    allowed for infinite sums, as it will be common for different numbers of
+    effective terms to be summed. Also beware of sums that may sum exactly to
+    zero; these may have significant numerical roundoff accumulation.
+    
+    When sums are performed over large numbers of terms, it's very easy for
+    numerical error to creep in. Make sure that your tolerance is sufficiently
+    large to satisfy such situations. When in doubt, use debug mode to view
+    what the sums are actually evaluating to.
+    
+    Configuration Options
+    =====================
+        answers (dict, required): Specifies author's answer. Has required keys lower,
+            upper, summand, summation_variable, which each take string values.
+
+        input_positions (dict): Specifies which integration parameters the student
+            is required to enter. The default value of input_positions is:
+
+                input_positions = {
+                    'lower': 1,
+                    'upper': 2,
+                    'summand': 3,
+                    'summation_variable': 4
+                }
+
+            and requires students to enter all four parameters in the indicated
+            order.
+
+            If the author overrides the default input_positions value, any subset
+            of the keys ('lower', 'upper', 'summand', 'summation_variable')
+            may be specified. Key values should be
+
+                - continuous integers starting at 1, or
+                - (default) None, indicating that the parameter is not entered by student
+
+            For example,
+
+                input_positions = {
+                    'lower': 1,
+                    'upper': 2,
+                    'summand': 3
+                }
+
+            indicates that the problem has 3 input boxes which represent the
+            lower limit, upper limit, and summand in that order. The
+            summation_variable is NOT entered by student and is instead the
+            value specified by author in 'answers'.
+
+        infty_val (int): Specifies a large number to be used in place of infinity in limits (default 1e3).
+                         Large values may cause timeouts.
+
+        infty_val_fact (int): Specifies a number to be used in place of infinity in limits when factorials
+                              are involved (default 100). Note that 100! ~ 10^157, which should be enough
+                              for most purposes!
+
+        even_odd (int): Choose to sum every number (0), every odd number (1) or every even number (2).
+
+    Additional Configuration Options
+    ================================
+    The configuration keys below are the same as used by FormulaGrader and
+    have the same defaults, except where specified
+        user_constants: same as FormulaGrader, but with additional default 'infty'
+        whitelist
+        blacklist
+        tolerance (default changed to 1e-12)
```

Comment:
This is in place because of the way that infinite sums are treated. If an infinite sum converges, then the terms should be getting smaller and smaller after some point. Because of the way that infinite sums can be rewritten, the comparison between the student and instructor answers may include different numbers of terms. The idea of an absolute tolerance is to allow for the different number of terms.

tl,dr; infinite sums are tricksy things and I think an absolute tolerance works better there.